src/custom_layers.py

```diff
@@ -0,0 +1,72 @@
+from keras.engine import Layer, InputSpec
+try:
+    from keras import initializations
+except ImportError:
+    from keras import initializers as initializations
+import keras.backend as K
+
+class Scale(Layer):
```

Comment:
Did you write this class by your own? If you didn't it's always a good idea to put the link where you found it in the document for later referencing :) . I'm a bit curious about why we need this class 🤔